File: nextjs-app/types/placed-student.ts
export interface PlacedStudent {
  id: number;
  student_id: string;
  certificate_id: string;
  student_name: string;
  student_image_url: string;
  company_name: string;
  student_role: string;
  student_bio: string;
  certificate_url: string;
  package?: string;
  location?: string;
  company_logo?: string;
  created_at?: string;
  updated_at?: string;
  is_active?: boolean;
}

File: fastapi_backend/schemas/placed_student.py
from pydantic import BaseModel
from typing import Optional
from datetime import datetime

class PlacedStudentItem(BaseModel):
    id: int
    student_id: str
    certificate_id: str
    student_name: str
    student_image: str
    student_image_url: str
    company_name: str
    student_role: str
    student_bio: Optional[str] = None
    certificate: Optional[str] = None
    certificate_url: Optional[str] = None
    is_active: bool
    created_at: Optional[datetime] = None
    updated_at: Optional[datetime] = None

    class Config:
        from_attributes = True
